### PR TITLE
Add Keepers Avalanche config

### PR DESCRIPTION
**Pull request by ritika-b**
- added info to Supported Networks page

### CODE REVIEW

File: docs/chainlink-keepers/supported-networks.html [hunk 1]
```diff
@@ -76,5 +80,37 @@
...
+    The maximum amount of gas that can be used by your checkUpkeep for off-chain computation.
+* **Call Gas Limit (callGasLimit):** <span>5,000,000</span> <!-- @TODO render this programatically. <span id="show-callGasLimit"></span> -->
+    The maximum amount of gas that can be used by the client contract’s performUpkeep for the on-chain transaction. You can set an upper limit on your Upkeep during registration, but this number must not exceed the call gas limit on the KeeperRegistry.
+* **Gas Ceiling Multiplier (gasCeilingMultiplier):** <span>2</span>
+    Establishes a ceiling for the maximum price based on the on-chain fast gas feed.
+
+{% endmarkdown %}
+
+{% markdown %}
+### Fantom
```
Comment (dwightjl) [inline docs/chainlink-keepers/supported-networks.html:100]: Let's move the Fantom changes into a separate PR.

File: docs/chainlink-keepers/supported-networks.html [hunk 2]
```diff
@@ -31,7 +31,11 @@
   - Mainnet
   - Testnet
 - Avalanche:
+  - Mainnet
```
Comment (dwightjl) [inline docs/chainlink-keepers/supported-networks.html:34]: Add Avalanche to the home page for Keepers: https://github.com/smartcontractkit/documentation/blob/c7e1e1aee48992eeb8d890dc4c9295fd30d66d17/docs/home.html#L273

**Review by dwightjl — CHANGES_REQUESTED** [review]

**Review by dwightjl — APPROVED** [review]

File: docs/chainlink-keepers/supported-networks.html (hunk 1 above)
Comment (dwightjl) [reply docs/chainlink-keepers/supported-networks.html]: Resolving.

File: docs/chainlink-keepers/supported-networks.html [hunk 3]
```diff
@@ -76,5 +77,21 @@
 
 {% endmarkdown %}
 
+{% markdown %}
+### Avalanche
+
+* **Payment Premium (paymentPremiumPPB):** <span>70</span>% gas premium
```
Comment (wentzeld) [inline docs/chainlink-keepers/supported-networks.html:83]: Payment Premium for Avalanche Fuji and Mainnet is 40% - this is the final amount

**Review by wentzeld — DISMISSED** [review]
Left some comments
1. Please change block count per turn to 200
2. Avax premium is 40%
3. Is there a config section for mainnet?

File: docs/chainlink-keepers/supported-networks.html [hunk 4]
```diff
@@ -76,5 +77,21 @@
...
+    This premium is applied on top of the gas cost paid by the Keeper to perform your Upkeep
+* **Block Count Per Turn (blockCountPerTurn):** <span>20</span>
```
[the remark this answers is not in the document]
Comment (dwightjl) [reply docs/chainlink-keepers/supported-networks.html]: Looks like this is fixed.

File: docs/chainlink-keepers/supported-networks.html (hunk 3 above)
Comment (dwightjl) [reply docs/chainlink-keepers/supported-networks.html]: This is fixed.

File: docs/chainlink-keepers/supported-networks.html [hunk 5]
```diff
@@ -31,6 +31,7 @@
   - Mainnet
   - Testnet
 - Avalanche:
+  - Mainnet
```
Comment (dwightjl) [inline docs/chainlink-keepers/supported-networks.html:34]:
```suggestion
  - Mainnet
  - Fuji Testnet
```

File: docs/chainlink-keepers/supported-networks.html [hunk 6]
```diff
@@ -31,6 +31,8 @@
...
+  - Fuji Testnet
```
Comment (dwightjl) [inline docs/chainlink-keepers/supported-networks.html:35]:
```suggestion
```

Comment (dwightjl) [reply docs/chainlink-keepers/supported-networks.html:35]: Accidentally added Fuji twice. Removing.

**Review by dwightjl — APPROVED** [review]
